src/comp/comp.py

```diff
@@ -24,48 +28,47 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans if human.name[0] in ['C', 'D', 'E', 'F', 'G']]
```

Comment:
very nice. nice. there´s also an ordinal relation between letters in Python, so
'A' < 'B' (is true)
'X' > 'Y' (returns false)
so the code below also works

c = [x.name for x in humans if x.name[0] >= 'C' and x.name[0] <= 'G']